README.md

```diff
@@ -1,56 +1,48 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] [OPTIONAL] Branch should have a name of a feature or action e.g. ```created_sidebar``` or ```added_comments_section```, you can skip this requirement in this PR but please in future consider implementing this